Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1565,15 +1249,6 @@ protected void OnConfigurationUpdated(object sender, EventArgs e)
                 requiresRestart = true;
             }
 
-            var currentCertPath = CertificateInfo == null ? null : CertificateInfo.Path;
```

Comment:
I'm noticing this touches a lot of certificate-related stuff but this isn't mentioned in the PR description. Can you comment on what removing these sections changes from a functional level?

Reply:
Agreed, will do